setup.py

```diff
@@ -133,7 +133,7 @@ def get_git_sha() -> str:
         "exasol": ["sqlalchemy-exasol>=2.1.0, <2.2"],
         "excel": ["xlrd>=1.2.0, <1.3"],
         "firebird": ["sqlalchemy-firebird>=0.7.0, <0.8"],
-        "gsheets": ["shillelagh[gsheetsapi]>=0.7.1, <0.8"],
+        "gsheets": ["shillelagh[gsheetsapi]>=1.0.3, <2"],
```

Comment:
any breaking changes here?

Reply:
No breaking changes, just fixes and features.